src/charm.py

```diff
@@ -49,29 +70,35 @@ def __init__(self, *args):
         self.framework.observe(self.on.update_runner_bin, self._on_update_runner_bin)
         self.framework.observe(self.on.stop, self._on_stop)
 
-        self.framework.observe(
-            self.on.check_runners_action, self._on_check_runners_action
-        )
-        self.framework.observe(
-            self.on.reconcile_runners_action, self._on_reconcile_runners_action
-        )
-        self.framework.observe(
-            self.on.flush_runners_action, self._on_flush_runners_action
-        )
+        self.framework.observe(self.on.check_runners_action, self._on_check_runners_action)
+        self.framework.observe(self.on.reconcile_runners_action, self._on_reconcile_runners_action)
+        self.framework.observe(self.on.flush_runners_action, self._on_flush_runners_action)
+
+    def _get_runner_manager(
+        self, token: Optional[str] = None, path: Optional[str] = None
+    ) -> Optional[RunnerManager]:
+        """Get a RunnerManager instance, or None if missing config.
 
-    def _get_runner_manager(self, token=None, path=None):
-        """Get a RunnerManager instance, or None if missing config."""
+        Args:
+            token (Optional[str]): GitHub personal access token to manager the runners with.
+                Defaults to None.
+            path (str): GitHub repository path in the format '<org>/<repo>', or the GitHub
+                organization name. Defaults to None.
+        """
```

Comment:
Our standard includes a description of return values, do you think it is applicable here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#docstrings